src/components/Activity/ActivityCard.tsx

```diff
@@ -180,16 +180,20 @@ const ActivityCard = ({
             )}
           </HStack>
           {wiki.categories.length && (
-            <NextLink href={`/categories/${wiki.categories[0].id}`} passHref>
-              <Text
-                display={{ base: 'none', md: 'block' }}
-                color="brand.500"
-                fontWeight="bold"
-                cursor="pointer"
-              >
-                {wiki.categories[0].title}
-              </Text>
-            </NextLink>
+            <HStack>
+              {wiki.categories?.map((category, i) => (
+                <NextLink key={i} href={`/categories/${category.id}`} passHref>
+                  <Text
```

Comment:
When we use `passHref`, the child must be an anchor tag.
```suggestion
                <NextLink key={i} href={`/categories/${category.id}`} passHref>
                  <Text
                    as="a"